### PR TITLE
Add status icon to Character endpoint #115

### PR DESCRIPTION
**Pull request by guzbarraf**
Add status icon to Character endpoint #115
(works with 50x50px images, functionality with css is commented)

### CODE REVIEW

**Comment by guzbarraf** [comment]
Hello @lelouchB , is the project going to continue?

**Review by lelouchB — COMMENTED** [review]
> 
> 
> Hello @lelouchB , is the project going to continue?



Have you made the suggested changes?

File: frontend/src/pages/Components/Body/_Card.js [hunk 1]
```diff
@@ -1,25 +1,44 @@
...
+function Card(props){
+  const statusCharacter = (status) => {
+    switch(status) {
+      case 'Alive':
+        return 'green';
+      case 'Dead':
+      case 'Deceased':
+        return 'red';
+      case 'Operational':
+        return 'yellow';
```
[the remark this answers is not in the document]
Comment (guzbarraf) [reply frontend/src/pages/Components/Body/_Card.js:15]: Hi @lelouchB, I'll take care of it

[the remark this answers is not in the document]
Comment (guzbarraf) [reply frontend/src/pages/Components/Body/_Card.js:15]: @lelouchB there are no status icons in the endpoint branch

[the remark this answers is not in the document]
Comment (lelouchB) [reply frontend/src/pages/Components/Body/_Card.js:15]: Yes, you have to add the icons.

The issue is just for creating or finding icons from the web and adding it to endpoint branch.

Thanks

**Pull request closed without merging**